### ISSUE TITLE
the loss is 0 when I train the model using your code

### ISSUE THREAD

**Issue by SophieZhou** [agjayant/caffe-Person-ReID#63]
Hi, @agjayant  when I train the model using your solver and model prototxt file by my data. The loss is 0 at the begining. I am not sure it is right.

I0816 16:21:20.639792 24829 solver.cpp:590] Iteration 2560, lr = 0.001
I0816 16:21:23.234153 24829 solver.cpp:243] Iteration 2580, loss = 0
I0816 16:21:23.234200 24829 solver.cpp:590] Iteration 2580, lr = 0.001
I0816 16:21:25.816555 24829 solver.cpp:243] Iteration 2600, loss = 0
I0816 16:21:25.816591 24829 solver.cpp:590] Iteration 2600, lr = 0.001
I0816 16:21:28.449795 24829 solver.cpp:243] Iteration 2620, loss = 0
I0816 16:21:28.449837 24829 solver.cpp:590] Iteration 2620, lr = 0.001
I0816 16:21:31.028082 24829 solver.cpp:243] Iteration 2640, loss = 0

**Comment by SophieZhou** [comment]
I think I know why, I did not shuffle the data before.

**Comment by SophieZhou** [comment]
HI,@agjayant  I found that, when I trained the model, the loss keep the same as margin value, I am not sure why.

**Comment by agjayant** [comment]
I hope that you are using pre-trained weights(on ImageNet) to initialize your network.
Also, make sure that your input image list is in correct format. Please refer to: https://github.com/agjayant/caffe-Person-ReID/issues/61#issuecomment-292889705
